### PR TITLE
Adds support for NEM(XEM) public key address format.

### PR DESCRIPTION
**Pull request by alexmb15**
Adds support for NEM(XEM) public key address format.
Added 'nem-sdk' library as a dependency.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -211,6 +212,18 @@ const bech32Chain = (name: string, coinType: number, prefix: string) => ({
   name,
 });
 
+function b32encodeXemAddr(data: Buffer): string {
+  return nemSdk.default.model.address.b32encode(nemSdk.default.utils.convert.hex2a(data));
+}
+
+function b32decodeXemAddr(data: string): Buffer {
+  if(!nemSdk.default.model.address.isValid(data)) {
+    throw Error('Unrecofnised address format');
```
Comment (Arachnid) [inline src/index.ts]: Typo: "unrecognised"

File: src/__tests__/index.test.ts [hunk 2]
```diff
@@ -175,6 +175,14 @@ const vectors: Array<TestVector> = [
       { text: 'cosmos1depk54cuajgkzea6zpgkq36tnjwdzv4afc3d27', hex: '6e436a571cec916167ba105160474b9c9cd132bd' }
     ],
   },
+  {
+    name: 'XEM',
+    coinType: 43,
```
Comment (Arachnid) [inline src/__tests__/index.test.ts]: This list is sorted by coin type - can you please add it accordingly?

File: src/index.ts [hunk 3]
```diff
@@ -245,6 +258,12 @@ const formats: IFormat[] = [
     encoder: tronweb.address.fromHex,
     name: 'TRX',
   },
+  {
```
Comment (Arachnid) [inline src/index.ts]: This list is sorted by coinType, can you please add it accordingly?

**Comment by alexmb15** [comment]
Done it.

**Pull request merged**